src/foam/nanos/auth/FilterBySpidService.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+   package: 'foam.nanos.auth',
+   name: 'FilterBySpidService',
+   extends: 'foam.nanos.auth.ProxyUniqueUserService',
+   flags: ['java'],
+
+  documentation: `Filter users by spid.`,
```

Comment:
No need for backquotes, just use ' instead.

Reply:
okay, thank you!